### PR TITLE
Ladders and Layers

### PR DESCRIPTION
**Pull request by APickledWalrus**
This PR isn't that big but has some features that should be added.

> Directional ladders (1.12 and older)
> Number of layers for snow layers (All versions)

### CODE REVIEW

File: decoration.sk [hunk 1]
```diff
@@ -122,8 +122,22 @@ decoratives before flattening:
...
 	[any] furnace¦s = unlit furnace, lit furnace
-	ladder¦s = minecraft:ladder
+	
+	ladder item¦s = minecraft:ladder
+	south[ward] facing ladder = minecraft:ladder {Damage:0}
+	north[ward] facing ladder = minecraft:ladder {Damage:3}
+	east[ward] facing ladder = minecraft:ladder {Damage:4}
+	west[ward] facing ladder = minecraft:ladder {Damage:5}
+	
 	snow layer¦s = minecraft:snow_layer
```
Comment (TheBentoBox) [inline decoration.sk:132]: How about adding a `(1|one) layer snow layer = minecraft:snow_layer {Damage:0}` here? That way they can specifically target a thin snow layer.

Additionally, I don't love the alias name here; "two layer snow layer" seems a bit repetitive. Maybe change it to "thick", such as `two thick snow layer`? Open to opinions.

File: decoration.sk [hunk 2]
```diff
@@ -255,7 +271,16 @@ decoratives after flattening:
...
-	snow layer¦s = minecraft:snow
+	
+	snow layer¦s = minecraft:snow_layer
+	(2|two) layer snow layer = minecraft:snow_layer[layers=2]
+	(3|three) layer snow layer = minecraft:snow_layer[layers=3]
+	(4|four) layer snow layer = minecraft:snow_layer[layers=4]
+	(5|five) layer snow layer = minecraft:snow_layer[layers=5]
+	(6|six) layer snow layer = minecraft:snow_layer[layers=6]
+	(7|seven) layer snow layer = minecraft:snow_layer[layers=7]
+	(8|eight) layer snow layer = minecraft:snow_layer[layers=8]
```
Comment (TheBentoBox) [inline decoration.sk]: Same as above, maybe use "thick" and add one for a specifically one layer thick one.

**Review by TheBentoBox — CHANGES_REQUESTED** [review]
Looks good overall, couple bits of simple feedback.

**Comment by APickledWalrus** [comment]
I think both changes are good so I did them both. Should be good now!

**Review by TheBentoBox — APPROVED** [review]

**Comment by TheBentoBox** [comment]
Looks great, thanks as always!

**Pull request merged**